### PR TITLE
Add version number to vsix name [#150]

### PR DESCRIPTION
**Pull request by atwoosnam**
- Add PS script to sync vsix version & build nums

### CODE REVIEW

File: build/update-version.ps1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$version = $args[0] -replace "-build", ""
```
Comment (ccheetham) [inline build/update-version.ps1:1]: Should we give attribution to the inspiration of this script?  E.g., a link to repo.

**Review by ccheetham — COMMENTED** [review]
see question

File: build/update-version.ps1 (hunk 1 above)
Comment (atwoosnam) [reply build/update-version.ps1:1]: Good call - I'll add a link now

**Review by ccheetham — APPROVED** [review]
lgtm

**Pull request merged**